packages/get-packages/package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@manypkg/find-root": "^1.1.0",
+    "@changesets/types": "*",
```

Comment:
That seems to be somewhat dangerous - we can't possibly guarantee that this will work with any potential version of this package. I would much more prefer if this would use the actual latest version with `^`. I know it forces us to bump this range and maintain over time - but at least it's safe.

Reply:
Can you suggest a version to use?

Reply:
```suggestion
    "@changesets/types": "^4.0.1",
```